.changes/raw-window-handle-std.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri": patch:deps
+---
+
+Add std feature to raw-window-handle crate so that no default features can work
```

Comment:
```suggestion
---
"tauri": patch:bug
---

Add `std` feature to `raw-window-handle` crate so that using `no-default-features` on `tauri` crate can work
```